backend/routes/recommendationHelpers.js

```diff
@@ -1,13 +1,28 @@
-const applyTimeDecay = (ratings) => {
+const applyTimeDecay = (allReviews) => {
+    const rateTime = {};
     const now = new Date();
-    return ratings.map(rating => {
-        const ageInDays = (now - new Date(rating.createdAt)) / (1000 * 60 * 60 * 24);
```

Comment:
One thing you can do here to make the code more readable. is pull some of the number out into const like:

`const SEC_IN_DAY = 60 * 60 * 24;`

then you can use it in your code like:

`const ageInDays = (now - new Date(rating.createdAt)) / (1000 * SEC_IN_DAY);`